feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.ts

```diff
@@ -153,17 +153,39 @@ export class ConfiguratorAttributeHeaderComponent
       .getConfiguration(this.owner)
       .pipe(take(1))
       .subscribe((configuration) => {
+        let groupId;
         if (this.groupType === Configurator.GroupType.CONFLICT_GROUP) {
-          const groupId = this.attribute.groupId;
-          if (groupId) {
-            this.configuratorGroupsService.navigateToGroup(
-              configuration,
-              groupId
-            );
-            this.focusAttribute(this.attribute.name);
-          }
+          groupId = this.attribute.groupId;
+        } else {
+          groupId = this.findConflictGroupId(configuration, this.attribute);
         }
+        if (groupId) {
```

Comment:
What happens if groupId is not defined i.e. group was not found?

Reply:
@ChristophHi: We will stay on the current page. This is the same behavior, as for the already implemented navigation from conflict to group (when the attribute is not found). Any other suggestion?

Reply:
Then I suggest to write a warning in case dev mode is active. This may help us and projects to identify issues. See e.g. ConfiguratorCartEntryBundleInfoService. (The situation might happen if the needed OCC enhancements are not present)